### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by tpulkit**
Added conda instructions separate from documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,20 +2,16 @@
...
-```
+- Create conda environment and install required packages:
+    ```
+    conda create --name myenv python=3.8.2
+    conda activate myenv
+    python -m pip install -r requirements.txt
+    ```
+- Add path to the repo to `PYTHONPATH`:
+    ```
+    export PYTHONPATH=$PYTHONPATH:$(pwd)
```
Comment (kedartatwawadi) [inline README.md:13]: maybe comment that you need to be in the repo for this to work
# cd <path to repo> 
or just 
`export PYTHONPATH=$PYTHONPATH:<path_to_repo>`

**Comment by kedartatwawadi** [comment]
@tpulkit: Can you confirm if all tests pass in the conda env, as I am not using that setup

File: README.md (hunk 1 above)
Comment (tpulkit) [reply README.md]: done

**Comment by tpulkit** [comment]
Yes, I have checked the whole process works and fixed your comment. Look at the latest commit.

**Review by kedartatwawadi — APPROVED** [review]
Looks good!

**Pull request merged**